active/0012-zone-based-auth.md

```diff
@@ -0,0 +1,30 @@
+# Authentication based on Zone
+
+## Changelog
+
+## Abstract
+
+We can define multiple zones with multiple authentication stack on emqx.
+
+## Motivation
+
+We have two types of clients, one of them connects using external network and needs to be authenticated by http
+but another group connects using an internal network and we need only jwt authentication for them.
+
+## Design
+
+## Configuration Changes
+
+There should be zone specific list of authentications.
```

Comment:
Could you clarify what the configuration should look like? In 5.0 we'll have the following hierarchy: 
```hocon
authorization:{
    rules: [
    ]
}
...
zone.x {
  a: {b:1, c: 1}
  listeners.y {
     a: {b: 2}
   }
}
```
Should we use `authentication` subtree as a default, with the possibility of adding zone-specific overrides?

Reply:
Yes, exactly. we can have exact same subtree under any zone.

Reply:
Could you please add the example config to the EIP?

Reply:
Yes, sure I will add it now.

Reply:
I have update it based on [emqx_authz](https://github.com/emqx/emqx/tree/master/apps/emqx_authz). Also, the new configuration is really great but I didn't have a chance to use it.

Reply:
Thanks, now the EIP looks clear to me.